addons/stock/models/stock_quant.py

```diff
@@ -255,25 +290,47 @@ def action_view_inventory(self):
         }
         return action
 
-    def action_apply_inventory(self):
-        quants_outdated = []
+    def action_apply_inventory_all(self, inventory_adjustment_name=None):
```

Comment:
Maybe you can directly call the wizard from javascript instead of creating a new function here